modules/performanceplatform/manifests/proxy_vhost.pp

```diff
@@ -53,6 +53,13 @@
 
   $graphite_servername = regsubst($servername, '\.', '_', 'G')
 
+  if ($serveraliases != undef) {
+    validate_array($serveraliases)
+    $vhost_names = $serveraliases
+  } else {
+    $vhost_names = [$servername]
```

Comment:
I think this will always evaluate to undef - if it is a string it will fail at the validate_array step